vllm/model_executor/layers/quantization/utils/fp8_utils.py

```diff
@@ -21,9 +21,7 @@
 
 logger = init_logger(__name__)
 
-current_platform_fp8_dtype = (torch.float8_e4m3fnuz
-                              if current_platform.is_rocm() else
-                              torch.float8_e4m3fn)
+current_platform_fp8_dtype = current_platform.fp8_dtype()
```

Comment:
I think it would actually be better to get rid of this and use `current_platform.fp8_dtype()` directly

Reply:
Ok, done. Removed it again and updated corresponding code.